examples/StandAloneListener/standalonelistener.py

```diff
@@ -1,11 +1,38 @@
 '''
-        This python script will listen to the defined vip address for specific
-    topics.  The user can modify the settings.py file to set the specific
-    topics to listen to.
 
```

Comment:
There should be a preamble before starting at 1.

Reply:
fixed